### DISCUSSION TITLE
Can you call a method that has 2 copies with the same name/param count?

### DISCUSSION THREAD

**Discussion by sanktanglia** [vfsfitvnm/frida-il2cpp-bridge#515] [Q&A] [answered]
Im currently trying to call this constructor but it doesnt seem to be working(object is undefined)

    [Token(Token = "0x6000C9F")]
    public Id(string key)
    {
    }

    [Token(Token = "0x6000CA0")]
    public Id(Id<T> thing)
    {
    }

**Comment by thinhbuzz** [comment, marked as answer]
Use overload like
method('Id').overload('System.String')
